Get travis working

Here is the travis output for master branch

```[0K$ make testacc
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test $(go list ./... |grep -v 'vendor') -v  -timeout 120m
client/client.go:5:2: cannot find package "context" in any of:
	/home/travis/.gimme/versions/go1.1.src/src/pkg/context (from $GOROOT)
	/home/travis/gopath/src/context (from $GOPATH)
nutanix/data_source_nutanix_clusters.go:10:2: cannot find package "github.com/hashicorp/terraform/helper/resource" in any of:
	/home/travis/.gimme/versions/go1.1.src/src/pkg/github.com/hashicorp/terraform/helper/resource (from $GOROOT)
	/home/travis/gopath/src/github.com/hashicorp/terraform/helper/resource (from $GOPATH)
nutanix/data_source_nutanix_clusters.go:11:2: cannot find package "github.com/hashicorp/terraform/helper/schema" in any of:
	/home/travis/.gimme/versions/go1.1.src/src/pkg/github.com/hashicorp/terraform/helper/schema (from $GOROOT)
	/home/travis/gopath/src/github.com/hashicorp/terraform/helper/schema (from $GOPATH)
main.go:4:2: cannot find package "github.com/hashicorp/terraform/plugin" in any of:
	/home/travis/.gimme/versions/go1.1.src/src/pkg/github.com/hashicorp/terraform/plugin (from $GOROOT)
	/home/travis/gopath/src/github.com/hashicorp/terraform/plugin (from $GOPATH)
nutanix/provider.go:5:2: cannot find package "github.com/hashicorp/terraform/terraform" in any of:
	/home/travis/.gimme/versions/go1.1.src/src/pkg/github.com/hashicorp/terraform/terraform (from $GOROOT)
	/home/travis/gopath/src/github.com/hashicorp/terraform/terraform (from $GOPATH)
flg/flg.go:13:2: cannot find package "github.com/spf13/pflag" in any of:
	/home/travis/.gimme/versions/go1.1.src/src/pkg/github.com/spf13/pflag (from $GOROOT)
	/home/travis/gopath/src/github.com/spf13/pflag (from $GOPATH)
flg/flg.go:14:2: cannot find package "github.com/spf13/viper" in any of:
	/home/travis/.gimme/versions/go1.1.src/src/pkg/github.com/spf13/viper (from $GOROOT)
	/home/travis/gopath/src/github.com/spf13/viper (from $GOPATH)
jsonToSchema/field.go:12:2: cannot find package "github.com/fatih/camelcase" in any of:
	/home/travis/.gimme/versions/go1.1.src/src/pkg/github.com/fatih/camelcase (from $GOROOT)
	/home/travis/gopath/src/github.com/fatih/camelcase (from $GOPATH)
make: *** [testacc] Error 1

travis_time:end:0ad2d7b4:start=1526355419096376085,finish=1526355419481905521,duration=385529436
[0K
[31;1mThe command "make testacc" exited with 2.[0m

Done. Your build exited with 1.
´´´

Any thoughts? @thetonymaster, @cristian?

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,13 +1,20 @@
 language: go
 go: 
-  - 1.10
+  - "1.10"
 
 go_import_path: github.com/terraform-providers/terraform-provider-nutanix
 
+before_install:
+  - curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
+  - dep status
+  - dep ensure
+
 install:
-- bash scripts/gogetcookie.sh
-- curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
+  - export GOPATH="${TRAVIS_BUILD_DIR}/vendor:$GOPATH"
+  - make build 
+
+before_script:
+  - make vet
 
 script:
-- make vendor-status
-- make testacc
+  - make testacc

--- a/nutanix/data_source_nutanix_clusters_test.go
+++ b/nutanix/data_source_nutanix_clusters_test.go
@@ -24,13 +24,6 @@ func TestAccOutscaleClustersDataSource_basic(t *testing.T) {
 
 // Lookup based on InstanceID
 const testAccClustersDataSourceConfig = `
-provider "nutanix" {
-  username = "admin"
-  password = "Nutanix/1234"
-  endpoint = "10.5.81.134"
-  insecure = true
-  port     = 9440
-}
 data "nutanix_clusters" "basic_web" {
 	metadata = {
 		length = 3

--- a/nutanix/data_source_nutanix_virtual_machine_test.go
+++ b/nutanix/data_source_nutanix_virtual_machine_test.go
@@ -30,14 +30,6 @@ func TestAccNutanixVMDataSource_basic(t *testing.T) {
 
 func testAccVMDataSourceConfig(r int) string {
 	return fmt.Sprintf(`
-provider "nutanix" {
-  username = "admin"
-  password = "Nutanix/1234"
-  endpoint = "10.5.81.134"
-  insecure = true
-  port     = 9440
-}
-
 variable clusterid {
   default = "000567f3-1921-c722-471d-0cc47ac31055"
 }

--- a/nutanix/data_source_nutanix_virtual_machines_test.go
+++ b/nutanix/data_source_nutanix_virtual_machines_test.go
@@ -24,14 +24,6 @@ func TestAccOutscaleVMSSDataSource_basic(t *testing.T) {
 
 // Lookup based on InstanceID
 const testAccVMSSDataSourceConfig = `
-provider "nutanix" {
-  username = "admin"
-  password = "Nutanix/1234"
-  endpoint = "10.5.81.134"
-  insecure = true
-  port     = 9440
-}
-
 variable clusterid {
   default = "000567f3-1921-c722-471d-0cc47ac31055"
 }

--- a/nutanix/resource_nutanix_virtual_machine_test.go
+++ b/nutanix/resource_nutanix_virtual_machine_test.go
@@ -74,14 +74,6 @@ func testAccCheckNutanixVirtualMachineDestroy(s *terraform.State) error {
 
 func testAccNutanixVMConfig(r int) string {
 	return fmt.Sprint(`
-provider "nutanix" {
-  username = "admin"
-  password = "Nutanix/1234"
-  endpoint = "10.5.81.139"
-  insecure = true
-  port     = 9440
-}
-
 variable clusterid {
   default = "000567f3-1921-c722-471d-0cc47ac31055"
 }

